.changeset/six-frogs-play.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/core-plugin-api': minor
+'@backstage/app-defaults': minor
+'@backstage/core-app-api': minor
+'@backstage/plugin-user-settings': minor
```

Comment:
This should probably be a patch and maybe we should split some changesets here as your description doesn't cover the specific changes made in the plugin.

Reply:
Ok, I can change it to patch but splitting changesets is something I am not familiar with, have to take a look how that works.